sound/hybrid_font.h

```diff
@@ -205,6 +205,7 @@ class HybridFont : public SaberBase, public Looper {
   RefPtr<BufferedWavPlayer> next_hum_player_;
   RefPtr<BufferedWavPlayer> swing_player_;
   RefPtr<BufferedWavPlayer> lock_player_;
+  RefPtr<BufferedWavPlayer> idle_player_;
```

Comment:
Can't we just re-use the hum player for this?

Reply:
Sure can! done.